website/src/controllers/api-v3/user.js

```diff
@@ -917,4 +917,61 @@ api.userRebirth = {
   },
 };
 
+/**
+ * @api {post} /user/block/:uuid blocks and unblocks a user
+ * @apiVersion 3.0.0
+ * @apiName BlockUser
+ * @apiGroup User
+ * @apiSuccess {}
+**/
+api.blockUser = {
+  method: 'POST',
+  middlewares: [authWithHeaders(), cron],
+  url: '/user/block/:uuid',
+  async handler (req, res) {
+    let user = res.locals.user;
+    common.ops.blockUser(user, req);
+    await user.save();
+    res.respond(200, {});
```

Comment:
respond with the result from the operation, here as well in the other two ops belop